cmd/rook/secret.go

```diff
@@ -114,6 +114,10 @@ func getSecret(cmd *cobra.Command, args []string) {
 		rook.TerminateFatal(errors.Wrapf(err, "failed to get secret %q", secretName))
 	}
 
+	if s == "" {
+		rook.TerminateFatal(errors.Errorf("secret key %q is empty", secretName))
```

Comment:
This seems to introduce a changed  behavior. Shouldn't that be mentioned in the commit message?